src/hotspot/share/gc/shared/oopStorage.hpp

```diff
@@ -99,12 +99,12 @@ class OopStorage : public CHeapObjBase {
   };
 
   // Locks _allocation_mutex.
-  // precondition: ptr != null.
+  // precondition: ptr != nullptr.
   EntryStatus allocation_status(const oop* ptr) const;
 
   // Allocates and returns a new entry.  Returns null if memory allocation
   // failed.  Locks _allocation_mutex.
-  // postcondition: result == null or *result == null.
+  // postcondition: result == null or *result == nullptr.
```

Comment:
Inconsistent mix of "null" and "nullptr".

Reply:
Thanks!